packages/taquito/src/estimate/rpc-estimate-provider.ts

```diff
@@ -100,7 +100,7 @@ export class RPCEstimateProvider extends Provider implements EstimationProvider
       opbytes,
       opOb: { branch, contents },
     } = await this.forge(op);
-    const operation: RPCRunOperationParam = {
+    const operation: RPCSimulateOperationParam = {
       operation: { branch, contents, signature: SIGNATURE_STUB },
```

Comment:
will we consider removing when simulating signature_stub?

Reply:
good point, removed.